### PR TITLE
[CU-31pc6c3] Add a building of composable-bridge-devnet docker image

### PR DESCRIPTION
**Pull request by andor0**
## Issue
https://app.clickup.com/t/31pc6c3

### CODE REVIEW

File: .github/workflows/build-test-deploy.yml [hunk 1]
```diff
@@ -856,51 +856,49 @@ jobs:
...
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      - run: |
+          # enable required features (see https://github.com/cachix/install-nix-action/issues/19)
+          mkdir -p ~/.config/nix
+          echo "system-features = kvm" >> ~/.config/nix/nix.conf
+      - run: |
+          cachix watch-exec --jobs 16 --compression-level $CACHIX_COMPRESSION_LEVEL composable-community nix -- build .#bridge-devnet-dali-container --no-update-lock-file --show-trace -L
+          docker load --input ./result
```
Comment (dzmitry-lahoda) [inline .github/workflows/build-test-deploy.yml:899]: i would prefer it builds commit hash too non on main. people may need branches to test. there is example on how to do it in this file (actaully hat 100% cp paste with just change of package and container name)

File: .nix/mmr-polkadot-bin.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+with pkgs;
+rustPlatform.buildRustPackage rec {
+  # HACK: break the nix sandbox so we can build the runtimes. This
+  # requires Nix to have `sandbox = relaxed` in its config.
+  # We don't really care because polkadot is only used for local devnet.
+  __noChroot = true;
+  name = "mmr-polkadot-v${version}";
+  version = "0.9.27";
+  src = fetchFromGitHub {
+    repo = "polkadot";
```
Comment (dzmitry-lahoda) [inline .nix/mmr-polkadot-bin.nix:11]: please make reuse of this stuff. share it with usual polkadot and replace hashes via nix. version, rev, hash, cargoSha256, name could be inputs to template.

File: code/parachain/node/src/command.rs [hunk 3]
```diff
@@ -41,7 +41,7 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, St
 		// Must define the default chain here because `export-genesis-state` command
 		// does not support `--chain` and `--parachain-id` arguments simultaneously.
 		#[cfg(feature = "dali")]
-		"dali-dev" => Box::new(chain_spec::dali_dev()),
+		id if id.contains("dali") => Box::new(chain_spec::dali_dev(id)),
```
Comment (dzmitry-lahoda) [inline code/parachain/node/src/command.rs:44]: not sure, what if we will have dali-rococo like others do tuned for rococo remote by parity?

Comment (dzmitry-lahoda) [reply code/parachain/node/src/command.rs:44]: that is one of the reason why we use nix - we share and compose.

**Review by KaiserKarel — APPROVED** [review]

**Comment by andor0** [comment]
@Mergifyio refresh

File: .github/workflows/build-test-deploy.yml (hunk 1 above)
Comment (andor0) [reply .github/workflows/build-test-deploy.yml:922]: done.

File: .nix/mmr-polkadot-bin.nix (hunk 2 above)
Comment (andor0) [reply .nix/mmr-polkadot-bin.nix]: done.

File: code/parachain/node/src/command.rs (hunk 3 above)
Comment (andor0) [reply code/parachain/node/src/command.rs:44]: then we will update the code for this case.

**Review by dzmitry-lahoda — APPROVED** [review]

**Pull request merged**